packages/agent-utils/src/scripts/js-engine/shims.ts

```diff
@@ -1,499 +1,31 @@
 import { globalToShimVarNameMap } from "./shims/globals";
```

Comment:
This file is generated via the rollup compilation step, see here: https://github.com/polywrap/evo.ninja/blob/30f6c6e6805d4a90e3b7570950e145740434c65c/packages/agent-utils/rollup.config.mjs#L7-L12

Reply:
So I don't think the work you've done here is necessary as the problem you initially outlined (duplicate code) isn't actually the case.

Reply:
So, the TS file with the JS string is auto-generated. We don't maintain it. The JS string that is injected into the JS engine needs to be a single bundled string; that's why we're bundling it with rollup